src/utils/BatchManager.js

```diff
@@ -153,7 +158,10 @@ class BatchManager {
       }
 
       return renderFn(context.props);
-    }).then((html) => {
+    }).then((html) => { // eslint-disable-line consistent-return
+      if (!html) {
+        return Promise.reject(noHTMLError);
```

Comment:
you'll want to create the error here, so the stack trace is correct

Reply:
I've set `stack` to null above actually so the stack trace doesn't even show. Showing the stack trace will likely confuse people since they'll believe that the error is coming from within Hypernova itself but the error is truly in their application somewhere.

Reply:
I'd love to do `process.on('unhandledRejection')` somewhere and catch potential unhandled promise rejections and then forward those over to the user but given that the process is shared between requests I don't think it's possible :(